release_build_files/readme.md

```diff
@@ -567,6 +567,11 @@ code.
 
 ## Release Notes
 
+### 8.1.0
+-   Changes
+    -   Firestore: Fixed a linker error when DocumentChange::npos was used.
```

Comment:
Nit: please wrap `DocumentChange::npos` in backticks.

Reply:
Done.